src/app/about/page.tsx

```diff
@@ -4,27 +4,43 @@ import Header from '../elements/header'
 export default function About() {
   return (
     <main className="min-h-screen p-12">
-      <Header/>
+      <Header />
 
       <div className="m-auto max-w-5xl w-full mt-12">
         <h1 className="text-2xl">
           Om projektet
         </h1>
 
         <p className="mt-2">
-          Som många andra, så har jag och sambon glömt att flytta bilen inför en städdag, vilket resulterade i saftiga böter.
-          Vi ville ha ett sätt att dels veta vart vi borde ställa bilen för att ha så lång tid som möjligt,
-          och gärna att kunna få påminnelser om bilen skulle behöva flyttas. 
+          Projektet är skapad av   <a
+            className="lg:pointer-events-auto pointer-events-none italic hover:underline"
+            href="https://www.johanehrenfors.se"
+            target="_blank"
+            rel="author"
+          >
+            Johan Ehrenfors&nbsp;  
+          </a>
+         med bidrag av&nbsp;
+          <a
+            className="lg:pointer-events-auto pointer-events-none italic hover:underline"
+            href="https://www.fullstackfrancis.com"
+            target="_blank"
+            rel="author"
+          >
+             Francis Jones
+          </a>, i Typescript och Next.js.
+          
         </p>
 
         <p className="mt-2">
-          Att bygga en sådan applikation tar en del tid, som ni säkert förstår - men jag tänkte börja
-          med att helt enkelt bygga en enkel karta som dynamiskt visar vart det är lämpligast att parkera.
+          Som många andra, så har vi glömt att flytta bilen inför en städdag, vilket har resulterat i saftiga böter.
+          Vi ville ha ett sätt att dels veta vart vi borde ställa bilen för att ha så lång tid som möjligt,
```

Comment:
Reading this sentence again, it would sound much better as "Vi ville snabbt kunna ta reda på vart vi kan låta bilen stå så länge som möjligt, men även ett smidigt sätt att få relevanta påminnelser om när vi måste flytta bilen igen." 

Totally my bad!